app/services/notifications/moderation/send.rb

```diff
@@ -0,0 +1,39 @@
+# Send notifications from moderation
+module Notifications
+  module Moderation
+    class Send
+      def initialize(available_moderators, notifiable)
+        @available_moderators = available_moderators
+        @notifiable = notifiable
+      end
+
+      def self.call(*args)
+        new(*args).call
+      end
+
+      delegate :user_data, :comment_data, to: Notifications
+
+      def call
+        # notifiable is currently only comment
+        moderator = available_moderators.sample
+        dev_account = User.dev_account
+        json_data = {
+          user: user_data(dev_account)
+        }
+        json_data[notifiable.class.name.downcase] = send "#{notifiable.class.name.downcase}_data", notifiable
```

Comment:
Using `send` is not a good practice, because it accesses private methods and it's safer not to do it in most situations.
Since we only support `Comment` moderation notifications right now, there could be just `comment_data`.
Or, if you want it to be used easier with other `notifiable` types in the future: 
- check the notifiable class to be included in the supported list (for now - only `Comment`)
e.g. using a guard clause at the beginning of the method and return if the notifiable class is not in the list
- use `public_send` instead of `send`

Reply:
I went for the public_send and updated the code.